addons/website_slides_survey/models/slide_channel.py

```diff
@@ -8,3 +8,23 @@ class Channel(models.Model):
     _inherit = 'slide.channel'
 
     nbr_certification = fields.Integer("Number of Certifications", compute='_compute_slides_statistics', store=True)
+
+    def _remove_membership(self, partner_ids):
+        res = super()._remove_membership(partner_ids)
+        user_inputs = self.env['survey.user_input'].sudo().search([
+            ('survey_id', 'in', self.slide_ids.survey_id.ids),
+            ('partner_id', 'in', partner_ids),
```

Comment:
shouldn't you also search for `('active', '=', True)` ? (to be consistent with `_action_add_members`)